proto/cluster.proto

```diff
@@ -166,34 +167,36 @@ message Cluster {
   Environment environment = 5;
   
   // Required field. This field indicates ray cluster configuration
-  ClusterSpec cluster_spec = 6;
+  ClusterSpec cluster_spec = 6 [(google.api.field_behavior) = REQUIRED];
+ 
   // Optional. Annotations, for example, "kubernetes.io/ingress.class" to define Ingress class 
-  map<string, string> annotations = 7;
+  map<string, string> annotations = 7 [(google.api.field_behavior) = OPTIONAL];
 
   // Optional input field. Container environment variables from user.
-  EnvironmentVariables envs = 8;
+  EnvironmentVariables envs = 8 [(google.api.field_behavior) = OPTIONAL];
 
   // Output. The time that the cluster created.
-  google.protobuf.Timestamp created_at = 9;
+  google.protobuf.Timestamp created_at = 9 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
True, its output only, but do we really need to mark it? Its optional by default. Even if it is specified on input, it will be ignored.

Reply:
@blublinsky it influences the generated swagger, it should be kept. Take a closer look at the generated swagger files.